src/pages/younseop/Main/Components/Feed.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useState } from 'react';
+import CommentBox from './Comments';
+
+const Feed = () => {
+  const [input, setInput] = useState('');
+  const [comment, setComment] = useState([]);
+
+  const handleInputChange = e => {
+    setInput(e.target.value);
+  };
```

Comment:
```suggestion
  const handleInputChange = e => {
    const { value } = e.target;
    setInput(value);
  };
```

destructuring 할당으로 보다 깔끔한 코드로 사용할 수 있습니다! (깔끔한지는 잘 모르겠지만..)